src/test/scala/edu/ie3/simona/model/participant/PvModelITSpec.scala

```diff
@@ -42,17 +47,16 @@ class PvModelITSpec extends Matchers with UnitSpec with PvModelITHelper {
           val calc = model
             .calculatePower(0L, voltage, ModelState.ConstantState, neededData)
             .p
-            .toMegawatts
 
-          val sol = resultsMap(dateTime)(modelId).toMegawatts
+          val sol = resultsMap(dateTime)(modelId)
 
-          math.abs(calc - sol)
+          (calc, sol)
         }
       }
 
-      testRes.forall { diff =>
-        diff < 1e-14
-      } shouldBe true
+      forAll(testRes) { case (calc, sol) =>
+        calc should approximate(sol)
+      }
```

Comment:
this is fine but could further reduced if you put 

`calc should approximate(sol)`

where you return the inner loop (calc,sol)